### PR TITLE
Update generic-host.md

### LINKED ISSUE #35891: .NET Generic Host - Code example uses CreateApplicationBuilder, text says CreateDefaultBuilder?
Hi,

in the "Set up a host" section, the text says "Calls a **CreateDefaultBuilder()** method ..." but in the code example it uses the newer **CreateApplicationBuilder()**.

I think the text should also be updated?

Ciao,
Mike

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d43ecd7c-307a-688e-d533-c1135bde65f9
* Version Independent ID: 22a12ff8-d2ff-a841-b9bb-e2fd1032bc04
* Content: [.NET Generic Host - .NET](https://learn.microsoft.com/en-us/dotnet/core/extensions/generic-host)
* Content Source: [docs/core/extensions/generic-host.md](https://github.com/dotnet/docs/blob/main/docs/core/extensions/generic-host.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @IEvangelist
* Microsoft Alias: **dapine**

### PR DESCRIPTION
**Pull request by IEvangelist**
Fixes #35891

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/extensions/generic-host.md](https://github.com/dotnet/docs/blob/c3f0a2021b0dfdc19191cf98fd815a7fba815aff/docs/core/extensions/generic-host.md) | [.NET Generic Host](https://review.learn.microsoft.com/en-us/dotnet/core/extensions/generic-host?branch=pr-en-us-35896) |

### CODE REVIEW

File: docs/core/extensions/generic-host.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ The main reason for including all of the app's interdependent resources in one o
 
 The host is typically configured, built, and run by code in the `Program` class. The `Main` method:
```
Comment (tdykstra) [inline docs/core/extensions/generic-host.md:27]: This is not necessarily something that needs to be corrected, but the reference to the Main method is out of date, as the snippet shows top level statements.

File: docs/core/extensions/generic-host.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ The main reason for including all of the app's interdependent resources in one o
...
 
-- Calls a <xref:Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder> method to create and configure a builder object.
+- Calls a <xref:Microsoft.Extensions.Hosting.Host.CreateApplicationBuilder> method to create and configure a builder object.
```
Comment (tdykstra) [inline docs/core/extensions/generic-host.md:29]: In the second instance the xref ends in %2A -- for consistency, we could add it here. The difference is that not adding it renders () after the method name.

```suggestion
- Calls a <xref:Microsoft.Extensions.Hosting.Host.CreateApplicationBuilder%2A> method to create and configure a builder object.
```

**Review by tdykstra — APPROVED** [review]
LGTM, a couple comments on preexisting issues.

File: docs/core/extensions/generic-host.md (hunk 1 above)
Comment (IEvangelist) [reply docs/core/extensions/generic-host.md:27]: I'll update this as part of a related issue I'm tracking [35897](https://github.com/dotnet/docs/issues/35897).

**Pull request merged**